libkbfs/init.go

```diff
@@ -169,6 +169,10 @@ func defaultLogPath(ctx Context) string {
 
 // DefaultInitParams returns default init params
 func DefaultInitParams(ctx Context) InitParams {
+	journalEnv := os.Getenv("KBFS_ENABLE_JOURNAL")
```

Comment:
Would a KBFS_DISABLE_JOURNAL make more sense?

Reply:
Agreed that DISABLE would be less confusing in the code.  Or "KBFS_DEFAULT_ENABLE_JOURNAL_VALUE"?